com.github.you_win.VPupPr.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id>com.github.youwin.VPupPr</id>
+  <launchable type="desktop-id">com.github.youwin.VPupPr.desktop</launchable>
+  <name>Virtual Puppet Project</name>
+  <summary>VTuber application made with Godot 3.4</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <releases>
+    <release>type="stable" version="0.8.1"</release>
+  </releases>
+  <description>
+    <p>VTuber application made with Godot 3.4.</p>
+    <p>This flatpak pulls osf-gd 0.8.1. When vpuppr is released, it will be updated accordingly.</p>
+  </description>
+  <keywords>
+    <keyword>vtuber</keyword>
+    <keyword>godot</keyword>
+    <keyword>face</keyword>
+    <keyword>tracking</keyword>
+    <keyword>renderer</keyword>
+  </keywords>
+  <url type="homepage">https://github.com/virtual-puppet-project/vpuppr</url>
+  <url type="bugtracker">https://github.com/virtual-puppet-project/vpuppr/issues</url>
+  <url type="donation">https://ko-fi.com/you_win</url>
+  <screenshots>
+    <screenshot type="default">putascreenshothere</screenshot>
+    <screenshot>putascreenshothere</screenshot>
+    <screenshot>putascreenshothere</screenshot>
```

Comment:
This is not valid. See the [specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-screenshots).

Reply:
I am very well aware of this. I plan on properly putting the screenshots there if/when the project is added to Flathub, just so that there's less hassle of having to deal with online image links and whatnot.

Reply:
That will not work. The AppStream file is validated on each build. If your AppStream is not valid, the App can't be added to Flathub. The workaround here is to use the URL of the Screenshots of your current working Repo and chance the URL after your App has been added to Flathub and you have your own Flathub Repo.

Reply:
Yes

Reply:
I've added one, as the other ended up crashing the app and I don't know if that would affect anything else. It should be ready to go now, I think?